src/lbp/lbp.rs

```diff
@@ -49,6 +50,38 @@ pub fn calculate_spot_price(
     to_balance!(spot_price)
 }
 
+fn convert_to_fixed(value: Balance) -> FixedBalance {
+    if value == Balance::from(1u32) {
+        return FixedBalance::from_num(1);
```

Comment:
What is the purpose of this `if` statement? Optimization?

Reply:
The result of converting one will be actually zero due to rounding as result of subsequent operations. 

Hoeweer, this part will be refactored and From/Into traits will be implemented for these types.